### DISCUSSION TITLE
Customizing UI - adding static assets such as images to ejs templates

### DISCUSSION THREAD

**Discussion by heaversm** [umputun/remark42#1669] [Q&A]
I want to be able to add images to the UI. I've got a backend and frontend instance going, but I'm not sure how to include images. I see some svgs in `assets/social` but they are being included via javascript requires. But how do I reference images in the assets folder for inclusion in the ejs files.

E.G. demo.ejs:

```
<body>
<img src="assets/social/twitter.svg"/>
...

wouldn't work. Also tried putting them in `site/public` where the favicons are, but that didn't seem to work (e.g referenced with `/image.png`)

**Comment by paskal** [comment]
Could you please describe your case? Technically, whatever is in `/srv/frontend/apps/remark42/public/` after the frontend Docker image build, will be [copied](https://github.com/umputun/remark42/blob/bbfa4f1/Dockerfile#L106) and served by backend.
